emscripten.py

```diff
@@ -631,7 +572,7 @@ def add_standard_wasm_imports(send_items_map):
     # the wasm backend reserves slot 0 for the NULL function pointer
     send_items_map['__table_base'] = '1'
   if shared.Settings.RELOCATABLE:
-    send_items_map['__stack_pointer'] = 'STACK_BASE'
+    send_items_map['__stack_pointer'] = "new WebAssembly.Global({value: 'i32', mutable: true}, STACK_BASE)"
```

Comment:
nice this is so easy to do!

Reply:
This actually end up landing as part of #12536